### PR TITLE
pkg/util/etcdutil: improve etcd migration logs

### PR DESCRIPTION
**Pull request by squat**
After running into issues deploying self-hosted etcd on Tectonic, I saw
lots of error logs. I thought the logs could be improved and be more
consistent.

cc @hongchaodeng

### CODE REVIEW

File: pkg/util/etcdutil/migrate.go [hunk 1]
```diff
@@ -53,19 +53,19 @@ func Migrate(kubeConfig clientcmd.ClientConfig, etcdServiceIP string) error {
...
 	if err := createMigratedEtcdCluster(restClient, ip); err != nil {
-		return fmt.Errorf("fail to create migrated etcd cluster: %v", err)
+		return fmt.Errorf("failed to create etcd cluster for migration: %v", err)
 	}
-	glog.Infof("etcd cluster for migration is created")
+	glog.Infof("created etcd cluster for migration")
 
 	if err := waitEtcdClusterRunning(restClient); err != nil {
-		return fmt.Errorf("wait etcd cluster running failed: %v", err)
+		return fmt.Errorf("failed to wait for etcd cluster to be running: %v", err)
```
Comment (hongchaodeng) [inline pkg/util/etcdutil/migrate.go]: "etcd cluster's status"

File: pkg/util/etcdutil/migrate.go [hunk 2]
```diff
@@ -158,7 +158,7 @@ func waitEtcdClusterRunning(restclient restclient.Interface) error {
 	err := wait.Poll(10*time.Second, waitEtcdClusterRunningTime, func() (bool, error) {
 		b, err := restclient.Get().RequestURI(makeEtcdClusterURI(etcdClusterName)).DoRaw()
 		if err != nil {
-			return false, fmt.Errorf("fail to get etcdcluster: %v", err)
+			return false, fmt.Errorf("failed to get etcd cluster: %v", err)
```
Comment (hongchaodeng) [inline pkg/util/etcdutil/migrate.go]: etcd cluster TPR

File: pkg/util/etcdutil/migrate.go [hunk 3]
```diff
@@ -195,12 +195,12 @@ func waitBootEtcdRemoved(etcdServiceIP string) error {
 		cancel()
 		etcdcli.Close()
 		if err != nil {
-			glog.Errorf("fail to list member, will retry: %v", err)
+			glog.Errorf("failed to list members, will retry: %v", err)
```
Comment (hongchaodeng) [inline pkg/util/etcdutil/migrate.go]: etcd members

**Comment by xiang90** [comment]
lgtm after addressing the minor issues hongchao pointed out. thanks!

**Comment by squat** [comment]
Thanks, @hongchaodeng and @xiang90. I addressed the nits. @yifan-gu or @pbx0 can someone with rights plz merge?

**Pull request merged**